<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="f415ecd3-164f-4e33-a307-ea2b462f8dd8" name="ivia_antonio@NOTE-ANTONIOJR 2012-01-12 11:54:32" runUser="ETTUSA\ivia_antonio" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="903c2975-19eb-4005-b9bb-37cf8b14a64f">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="ivia_antonio_NOTE-ANTONIOJR 2012-01-12 11_54_32">
      <DeploymentItem filename="C:\Ivia\dojo\TestabilityEF4\packages\EntityFramework.4.2.0.0\lib\net40\EntityFramework.dll" />
      <DeploymentItem filename="C:\Ivia\dojo\TestabilityEF4\Dominio\bin\Debug\Dominio.dll" />
      <DeploymentItem filename="C:\RedeInova\MyOrbe\RedeInova.Comum\bin\Debug\RedeInova.Comum.dll" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2012-01-12T11:54:32.3491752-03:00" queuing="2012-01-12T11:54:32.4361752-03:00" start="2012-01-12T11:54:32.4681752-03:00" finish="2012-01-12T11:54:33.0081752-03:00" />
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="Seu_Email_Deve_Ser_Unico" storage="c:\ivia\dojo\testabilityef4\dominio.tests\bin\debug\dominio.tests.dll" id="13aed554-06ec-83fb-856d-36601abacd57">
      <Execution id="e047f8a8-2cc1-4ccc-b013-40414b9f16ba" />
      <TestMethod codeBase="C:/Ivia/dojo/TestabilityEF4/Dominio.Tests/bin/Debug/Dominio.Tests.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="Dominio.Tests.Ao_Criar_Um_Usuario, Dominio.Tests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="Seu_Email_Deve_Ser_Unico" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="13aed554-06ec-83fb-856d-36601abacd57" executionId="e047f8a8-2cc1-4ccc-b013-40414b9f16ba" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="e047f8a8-2cc1-4ccc-b013-40414b9f16ba" testId="13aed554-06ec-83fb-856d-36601abacd57" testName="Seu_Email_Deve_Ser_Unico" computerName="NOTE-ANTONIOJR" duration="00:00:00.0974583" startTime="2012-01-12T11:54:32.5441752-03:00" endTime="2012-01-12T11:54:32.9801752-03:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="e047f8a8-2cc1-4ccc-b013-40414b9f16ba">
      <Output>
        <ErrorInfo>
          <Message>Test method Dominio.Tests.Ao_Criar_Um_Usuario.Seu_Email_Deve_Ser_Unico threw exception: 
System.NullReferenceException: Object reference not set to an instance of an object.</Message>
          <StackTrace>    at Dominio.Servicos.GerenteUsuarios.EmailJaExiste(String email) in C:\Ivia\dojo\TestabilityEF4\Dominio\Servicos\GerenteUsuarios.cs:line 43
   at Dominio.Servicos.GerenteUsuarios.ValidarNovoUsuario(Usuario novoUsuario) in C:\Ivia\dojo\TestabilityEF4\Dominio\Servicos\GerenteUsuarios.cs:line 30
   at Dominio.Servicos.GerenteUsuarios.CriarNovoUsuario(String email) in C:\Ivia\dojo\TestabilityEF4\Dominio\Servicos\GerenteUsuarios.cs:line 22
   at Dominio.Tests.Ao_Criar_Um_Usuario.Seu_Email_Deve_Ser_Unico() in C:\Ivia\dojo\TestabilityEF4\Dominio.Tests\UsuarioTests.cs:line 39
</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>